lib/workers/repository/dependency-dashboard.ts

```diff
@@ -220,15 +241,36 @@ function getBranchesListMd(
   if (filteredBranches.length === 0) {
     return '';
   }
-  let result = `## ${title}\n\n${description}\n\n`;
-  result += `${filteredBranches
-    .map((branch: BranchConfig): string => getListItem(branch, listItemType))
-    .join('')}`;
+
+  const { categories, uncategorized } =
+    splitBranchesByCategory(filteredBranches);
+  let result = `## ${title}\n\n${description}`;
+  if (Object.keys(categories).length > 0) {
+    for (const [category, branches] of Object.entries(categories).sort(
+      ([keyA], [keyB]) => keyA.localeCompare(keyB),
```

Comment:
```suggestion
      ([keyA], [keyB]) => keyA.localeCompare(keyB, undefined, {numeric: true}),
```

so it sorts number: `1<2<10` instead of `1<10<2`